src/components/NacelleLinker.js

```diff
@@ -44,6 +43,7 @@ const NacelleData = ({ dataType, active }) => {
           options={spaceOptions}
           className="tabContent"
           active={active}
+          type="products"
```

Comment:
Added a type prop here since the way of getting the results was a little confusing.